app/views/user_notifications/digest/_popular_posts.text.erb

```diff
@@ -1,10 +1,9 @@
 <%- if @popular_posts.present? %>
-### <%=t 'user_notifications.digest.popular_posts' %>
```

Comment:
does it matter that you are removing `###`

Reply:
this is a great question. I just checked, and it's part of the section headers (I assume for markdown). Added back in!